packages/devtools_server/lib/src/server.dart

```diff
@@ -458,6 +423,119 @@ Future<HttpServer> serveDevTools({
   return server;
 }
 
+ArgParser _createArgsParser(bool verbose) {
+  final parser = ArgParser();
+
+  parser
+    ..addFlag(
+      argHelp,
+      negatable: false,
+      abbr: 'h',
+      help: 'Prints help output.',
+    )
+    ..addFlag(
+      argVerbose,
+      negatable: false,
+      abbr: 'v',
+      help: 'Output more informational messages.',
+    )
+    ..addOption(
+      argHost,
+      //defaultsTo: 'localhost',
+      valueHelp: 'host',
+      help: 'Hostname to serve DevTools on (defaults to localhost).',
+    )
+    ..addOption(
+      argPort,
+      defaultsTo: '9100',
+      valueHelp: 'port',
+      help: 'Port to serve DevTools on; specify 0 to automatically use any '
+          'available port.',
+    )
+    ..addFlag(
+      argLaunchBrowser,
+      defaultsTo: true,
+      help: 'Launches DevTools in a browser immediately at start.',
+    )
+// TODO: Remove this - prefer that clients use the rest arg.
+    ..addOption(
+      argVmUri,
```

Comment:
@terrylucas - where is this option currently used? Somewhere on the flutter engine bots?